conftest.py

```diff
@@ -291,6 +283,12 @@ def pytest_generate_tests(metafunc):
 
 
 def pytest_addoption(parser):
+    # Method to add the option to ini
```

Comment:
@nilaya123 Can you please convert it into docstring for example: " Method to add ..........." Check the full file and convert method description to docstring.

Reply:
@rohandudam I have done changes suggested..PLease help to review and merge..